.nvmrc

```diff
@@ -0,0 +1 @@
+10.13.0
```

Comment:
Since version should be locked (based on README comment), this is to ensure contributors are using the correct node version